docker/Dockerfile

```diff
@@ -9,11 +9,11 @@ ARG OS_VERSION=bookworm
 ARG FFMPEG_PACKAGE=jellyfin-ffmpeg7
 
 # https://github.com/intel/compute-runtime/releases
-ARG GMMLIB_VER=22.7.0
+ARG GMMLIB_VER=22.8.1
 # >= Gen12 graphics (current)
-ARG IGC2_VER=2.10.8
-ARG IGC2_BUILD=18926
-ARG NEO_VER=25.13.33276.16
+ARG IGC2_VER=2.18.5
+ARG IGC2_BUILD=19820
+ARG NEO_VER=25.35.35096.9
```

Comment:
I remember that they removed support for Glibc prior to Ubuntu 24.04 in recent versions, which means they can't be installed on Bookworm.

As for .NET 9 support on Trixie, MS said they have no plans and will jump directly to .NET 10. And they tend to use Ubuntu. https://github.com/dotnet/dotnet-docker/issues/6620